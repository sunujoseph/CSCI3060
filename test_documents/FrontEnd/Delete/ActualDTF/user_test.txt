02 userFS AA 000000000
