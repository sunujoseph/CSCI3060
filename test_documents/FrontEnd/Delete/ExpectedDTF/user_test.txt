02 userFS          FS 111111.11
00 admin1          AA 000150.78
