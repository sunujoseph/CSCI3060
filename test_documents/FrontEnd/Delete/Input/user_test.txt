login
admin1
delete
userFS
