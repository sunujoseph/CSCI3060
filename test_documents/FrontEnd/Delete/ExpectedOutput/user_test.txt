Enter user name: 
Enter a user to delete the account of: 

